client/packages/core/src/Reactor.js

```diff
@@ -1817,7 +1863,27 @@ export default class Reactor {
     }
   }
 
+  async syncUserToEndpoint(user) {
+    if (this.config.endpointURI) {
```

Comment:
Curious idea: 

Do you guys think there's a way to make it so the user doesn't have to provide this by default. 

Perhaps we could do an attempt on `/instant/sync-auth` for example, and if it's good, we continue on.

Reply:
Hmm yeah if there is a 404 warning in console would not be good. If we used a HEAD request would it still log it as an issue?

Reply:
you still get errors

Reply:
SG!